libr/core/cmd_section.c

```diff
@@ -232,10 +264,10 @@ static int cmd_section(void *data, const char *input) {
 		}
 		break;
 	case '\0':
-		r_io_section_list (core->io, core->offset, 0);
+		print_sections (core->bin, core->offset, R_FALSE);
```

Comment:
this function is *very* similar to r_io_section_list but takes the the sections from RBin, not from RIO.